tests/unit/components/test.js

```diff
@@ -0,0 +1,57 @@
+var sd = require('skin-deep'),
+    expect = require('chai').expect,
+    Test = require('../../../ui/components/test.jsx'),
+    fixture = require('../../fixtures/components/test.js');
+
+describe('Test', function() {
+  var tree, passTest, failTest;
+
+  before(function() {
+    passTest = fixture[0];
+    failTest = fixture[1];
+  });
+
+  beforeEach(function() {
+    tree = sd.shallowRender(<Test test={passTest}/>);
+  });
+
+  it('should display the test title', function() {
+    expect(tree.textIn('.test-title')).to.contain(passTest.title);
+  });
+
+  it('should display the pass state', function() {
+    expect(tree.textIn('.test-state')).to.be.equal(passTest.state);
+  });
+
+  it('should display the fail state', function() {
+    tree = sd.shallowRender(<Test test={failTest}/>);
```

Comment:
Split this into a separate test file, or at least into a separate describe with another `beforeEach` block.